openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -0,0 +1,436 @@
+            raise ValueError("Must be temperature value, e.g. use unit.kelvin")
+
+    @validator('timestep', 'collision_rate')
+    def is_time(cls, v):
+        # these are time units, not simulation steps
+        if not v.is_compatible_with(unit.picosecond):
+            errmsg = "timestep and collision_rate must be in time units"
+            raise ValueError(errmsg)
+        return v
+
+
+class SimulationSettings(BaseModel):
+    """Settings for simulation control, including lengths, writing to disk,
+       etc...
+
+    Attributes
+    ----------
+    minimization_steps : int
+      Number of minimization steps to perform. Default 10000.
+    equilibration_length : float * unit.picosecond
+      Length of the equilibration phase in units of time.
+    production_length : float * unit.picosecond
+      Length of the production phase in units of time.
+    output_filename : str
+      Path to the storage file for analysis. Default 'rbfe.nc'.
+    checkpoint_interval : int * unit.timestep
+      Frequency to write the checkpoint file. Default 50 * unit.timestep
+    checkpoint_storage : str
+      Optional separate filename for the checkpoint file. Note, this should
+      not be a full path, just a filename. If None, the checkpoint will be
+      written to the same file as output_filename. Default None.
+    """
+    class Config:
+        arbitrary_types_allowed = True
+
+    minimization_steps = 10000
+    equilibration_length: unit.Quantity
+    production_length: unit.Quantity
+
+    # reporter settings
+    output_filename = 'rbfe.nc'
+    checkpoint_interval = 50 * unit.timestep
+    checkpoint_storage = Union[str, None]
+
+    @validator('equilibration', 'production')
+    def is_time(cls, v):
+        # these are time units, not simulation steps
+        if not v.is_compatible_with(unit.picosecond):
+            raise ValueError("Durations must be in time units")
+        return v
+
+    @validator('minimization', 'equilibration', 'production',
+               'checkpoint_interval')
+    def must_be_positive(cls, v):
+        if v <= 0:
+            errmsg = ("Minimization steps, MD lengths, and checkpoint "
+                      "intervals must be positive")
+            raise ValueError(errmsg)
+        return v
+
+
+class RelativeLigandTransformSettings(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    # Things for creating the systems
+    system_settings: SystemSettings
+    topology_settings: TopologySettings
+
+    # Alchemical settings
+    alchemical_settings: AlchemicalSettings
+
+    # MD Engine things
+    engine_settings: OpenMMEngineSettings
+
+    # Sampling State defining things
+    integrator_settings: IntegratorSettings
+    barostat_settings: BarostatSettings
+    sampler_settings: SamplerSettings
+
+    # Simulation run settings
+    simulation_settings: SimulationSettings
+
+    # solvent model?
+    solvent_padding = 1.2 * unit.nanometer
```

Comment:
Wasn't really keeping up with what the current plan is re: solvent - I've left this in for now.